app/routes.js

```diff
@@ -42,9 +42,11 @@ router.get('/solutions', (req, res) => {
 
 router.post('/solutions/capabilities-selector', withCatch(async (req, res) => {
   let capabilitiesSelected = req.body.capabilities;
+  if (!capabilitiesSelected) capabilitiesSelected = 'all';
   if (!Array.isArray(capabilitiesSelected)) capabilitiesSelected = [capabilitiesSelected];
-  const capabilitiesParam = capabilitiesSelected.join('+');
-  res.redirect(`/solutions/capabilities-selector.${capabilitiesParam}`);
+  const capabilitiesParam = `.${capabilitiesSelected.join('+')}`;
```

Comment:
can we move the above lines to a helper function that works out the capabilitiesParam. Might also get rid of that `let`

Reply:
I've moved it but there is still a let because the way the value is worked out is complex